main/main.js

```diff
@@ -146,8 +146,11 @@ app.on('ready', () => {
   // only set the menu template when in production mode/
   // this also leaves the dev console enabled when in dev mode.
   if (!process.defaultApp) {
-    const menu = Menu.buildFromTemplate(menuTemplate);
-    Menu.setApplicationMenu(menu);
+    const templatePromise = createMenuTemplate();
+    templatePromise.then((template) => {
```

Comment:
maybe we don't need to re-assign the promise? So like this instead?

```suggestion
    createMenuTemplate().then((template) => {
```